app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -274,10 +282,15 @@ private void startRecording() {
     private void toggleButtonsRecordingState() {
         boolean isInRecordingState = inRecordingState(this);
         swRecordingMaster.setChecked(isInRecordingState);
+
+        swGPSRecord.setEnabled(!isInRecordingState);
+        swGPSPrecision.setEnabled(!isInRecordingState);
```

Comment:
Is dit om database redenen dat het niet tijdens het recorden aangepast kan worden?

Reply:
Neen. Verzoek van PO om ze te blokken